### PR TITLE
Tidy up and extension of field activities (work records)

### PR DESCRIPTION
**Pull request by cookeac**
* Restructure measurements to have a base type (`MeasurementType`), and specific concrete types that specify unit enumerations.
* Added volume and mass units for product applications
* Add solar irradiance type and remove climate measurement type
* Move `OperationRecordResource` and `WorkRecordResource` to the resources folder
* Lint and correct some typos

### CODE REVIEW

File: types/MeasurementType.json [hunk 1]
```diff
@@ -1,18 +1,14 @@
 {
     "$schema": "https://json-schema.org/draft/2020-12/schema",
-    "$id": "/types/QuantityMeasurementType",
-    "description": "Quantity measures",
+    "$id": "/types/AreaMeasureType",
```
Comment (danielR9) [inline types/MeasurementType.json]: Don't think this id is correct if this is the base measurement type?

File: types/SpatialMeasureType.json [hunk 2]
```diff
@@ -0,0 +1,21 @@
+{
+    "description": "Specifies a spatial measurement such as area or length, including units (default is m2). Express 1 hectare as 10,000 m2.",
+
```
Comment (danielR9) [inline types/SpatialMeasureType.json:3]: Missing `$id`?

**Review by danielR9 — CHANGES_REQUESTED** [review]
The movement of types to resources has not changed the `$id` of them. So `resources/WorkRecordResource.json` has an id with `types/WorkRecordResource.

Same for other types that have been moved to resources directory

There is also inconsistence in the id naming conventions some are `types/IrradianceMeasureType` and others are `types/humidityMeasure`. Both with case, and the use of `Type` as a suffix

**Comment by cookeac** [comment]
@danielR9 I've corrected the $id values (all I could see), but I recall why the older Geospatial files and the newer ones I've created don't have the "$schema" and "$id" values.

We incorporate these into OAS3 OpenAPI (Swagger) files and OAS3 only seems to allow one $schema declaration per OAS3 file. This means any linting tool for OAS3 will give errors if each of the files included with $ref also contain a schema declaration.

I don't know for sure why it doesn't like $id, but similarly OAS3 linting tools seem to think you use it as a base for resolving local relative references, and give an error if you have then in each file. 

Do we use these for anything in any build processes? Otherwise I think we're better to remove them, and just specify the $schema in the example OAS3 URL schemes.

File: resources/WorkRecordResource.json [hunk 3]
```diff
@@ -24,9 +22,13 @@
         {
             "$ref":"../enums/WorkItemPriorityType.json"
         },
-        "area": {
+        "workedArea": {
             "$ref": "../types/AreaMeasureType.json",
-            "description": "Area of the operation"
+            "description": "Area worked by the operation"
+        },        
+        "nonOverlapArea": {
```
Comment (chriiiish) [inline resources/WorkRecordResource.json]: Consider renaming this?
```suggestion
        "nonOverlapWorkedArea": {
```

**Pull request merged**